src/app/infrastructure/storage/how-to-use-storage/upload-files-to-ipfs/page.mdx

```diff
@@ -0,0 +1,242 @@
+import {
+	Steps,
+	Step,
+	Carousel,
+	ArticleCard,
+	Callout,
+} from "@/components/Document";
+import Image from "next/image";
+
+
+import { Tabs, TabsList, TabsTrigger, TabsContent } from "@/components/ui/tabs";
+
+# Upload Files to IPFS
+
+Upload and pin files and directories directly to IPFS using the dashboard, CLI or via a client or server environment using SDKs.
+
+## Upload files to IPFS using dashboard
+
+To upload files using dashboard:
+
+<Steps>
+
+<Step title="Connect Wallet">
+To authenticate to your account, connect your wallet.
+</Step>
+
+<Step title="Storage Dashboard">
+Navigate to the [storage dashboard](https://thirdweb.com/dashboard/infrastructure/storage)
+
+   <Image
+	src={require("./assets/storage-dashboard.png")}
+	alt="storage diagram"
+    />
+
+</Step>
+
+<Step title="Select files">
+Select your files or drag them into the upload box. Confirm that your files are correct, and then select "Start Upload.”
+
+    <Image
```

Comment:
you don't have to indent content inside step component

Reply:
i think I did it to just make it more clear on the markdown what is inside of a step

Reply:
will change though